app/api/route.ts

```diff
@@ -1,56 +1,182 @@
-import { NextResponse } from "next/server";
+import { NextRequest, NextResponse } from "next/server";
+import { Event } from "../types/Event";
+import { mockEventJSON } from "../data/events";
+import { organizationsStore } from "../data/organizations";
 
+async function fetchEvent(organization: string) {
+  // Make sure it's a real organization
+  if (!(organization in organizationsStore)) {
```

Comment:
I don't this will work for users who are submitting "one off" events (as they may not have an organization) but we still want their event to show up.

Reply:
This just makes sure that the URL API request is an actual valid organization IIRC.

Reply:
Do we have a calendar setup for one-off events yet?
If we do, I should add it to the display all events thing (aka on /calendar) so that one-off events will be shown on the main page.

Reply:
We don't, but I suggest we make a separate issue so this PR isn't massive.